scripts/discover_devices.py

```diff
@@ -59,24 +63,49 @@
     nargs="?",
     default=60,
 )
+parser.add_argument(
+    "-t",
+    "--type",
+    required=False,
+    choices={1, 2},
+    help="set protocol type: 1 or 2",
```

Comment:
I would list here the the DeviceType.value of each type/category
"set protocol type: 
        1 [Switcher Mini, Switcher Power Plug, etc..] 
        2[Breeze, Runner, Runner Mini]

Reply:
and also I think --type somehow conflict with --ports
I think we should have one.

I think we should have single option to control the bridge socket:
--device-category [cat1, cat2, all]

Reply:
agree, so I will remove the option to configure the ports, I also thought at start to call it device category but it conflicts with the category we already have in the code. So maybe we keep it as `protocol type`.

I will remove the option to set the port and keep only the option for protocol type, e.g.:
`type1, type2, all`

And add text which explains which devices use each type (as you wrote above)